.github/workflows/build-and-push-docker.yml

```diff
@@ -0,0 +1,39 @@
+name: Docker Build and Push
+
+# Controls when the action will run. 
+# Triggers the workflow for a push to dev
+# with changes in Dockerfile.
+on:
+  push:
+    branches: [ dev ]
+    paths: [ Dockerfile ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build_docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      -
+        name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: kfdrc/open-pedcan:latest
```

Comment:
As long as you are ok losing old versions of the docker image this is fine.

Reply:
I think we are ok with losing the old version for this docker image